arangod/RocksDBEngine/RocksDBHotBackup.cpp

```diff
@@ -997,6 +998,22 @@ void RocksDBHotBackupRestore::execute() {
 
       restartAction = new std::function<int()>();
       *restartAction = localRestoreAction;
+
+      // Now remove all local ArangoSearch view data, since it will not be
+      // valid after the restore. Note that on a single server there is no
+      // automatism to recreate the data and on a dbserver, the Maintenance
+      // is stopped before we get here.
+      iresearch::IResearchFeature* arangoSearchFeature =
```

Comment:
honestly I don't understand why do we need feature instance for this. Local helper function which drops/recreates view is more than enough.